### ISSUE TITLE
FlatFileTarget IndexOutOfBoundException

### ISSUE THREAD

**Issue by kenpusney** [rec-framework/rec-core#36]
code:
```javascript
const cnt = counter(function() true);
const file = flat('timestamps.csv');

csv('combined.csv', "...")
   .tee(cnt)
   .to(file);
```
file
```
blabal, ... ,,,
```
```
Caused by: java.lang.IndexOutOfBoundsException: Index: 13, Size: 13
        at java.util.ArrayList.rangeCheck(Unknown Source)
        at java.util.ArrayList.get(Unknown Source)
        at net.kimleo.rec.sepval.SepValEntry.get(SepValEntry.java:19)
        at net.kimleo.rec.sepval.SepValEntry.get(SepValEntry.java:7)
        at net.kimleo.rec.v2.accessor.RecordWrapper.getByIndex(RecordWrapper.java:41)
        at net.kimleo.rec.v2.accessor.RecordWrapper.get(RecordWrapper.java:34)
        at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source)
        at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)
        at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
        at java.util.stream.ReferencePipeline.collect(Unknown Source)
        at net.kimleo.rec.v2.model.impl.FlatFileTarget.put(FlatFileTarget.java:30)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
        at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
        at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
        at java.util.Iterator.forEachRemaining(Unknown Source)
        at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Unknown Source)
        at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
        at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
        at java.util.stream.ReferencePipeline.forEach(Unknown Source)
        at net.kimleo.rec.v2.model.impl.FlatFileTarget.putAll(FlatFileTarget.java:35)
        at net.kimleo.rec.v2.model.Source.to(Source.java:23)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:126)
        ... 15 more
```

**Comment by kenpusney** [comment]
Issue occurred when last element was not provided but with an accessor.

**Comment by kenpusney** [comment]
Fixed in #37
